snapshots/education/2023-08-09/years_of_education_gender.xlsx.dvc

```diff
@@ -0,0 +1,20 @@
+meta:
+  name: Historical estimates of gender equality of numeracy, 1850 -2000. (OECD, 2013)
+  publication_year: 2013
+  publication_date: '2013-03-18'
+  source_name: Clio Infra
+  source_published_by: van Leeuwen, B. and van Leeuwen-Li, J. (2013). Educational
+    Inequality Gini Coefficient via Clio Infra
+  url: https://clio-infra.eu/Indicators/EducationalInequalityGiniCoefficient.html
```

Comment:
I think the url is not the correct one for this dataset, looks like it is a repeat of `years_of_education_gini`

Reply:
thanks a lot for spotting this!!